### PR TITLE
Fix sf::View documentation

### PR DESCRIPTION
**Pull request by Intemporel**
I just update the description for `sf::View - setViewport(const sf::FloatRect&)`

### CODE REVIEW

**Review by ChrisThrasher — DISMISSED** [review]
Thanks! Once another maintainer can look at this we should be able to merge it.

While you're at it, can you try to look for any other places where the comments reference this old `FloatRect` constructor? It's plausible we made this mistake elsewhere as well.

**Review by Bambo-Borris — APPROVED** [review]
Love to see another person throw some help SFML3's way!

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

File: include/SFML/Graphics/View.hpp [hunk 1]
```diff
@@ -109,7 +109,7 @@ class SFML_GRAPHICS_API View
     /// view are displayed, expressed as a factor (between 0 and 1)
     /// of the size of the RenderTarget to which the view is applied.
     /// For example, a view which takes the left side of the target would
-    /// be defined with View.setViewport(sf::FloatRect(0, 0, 0.5, 1)).
+    /// be defined with View.setViewport(sf::FloatRect({ 0.f, 0.f }, { 1.f, 1.f })).
```
Comment (kimci86) [inline include/SFML/Graphics/View.hpp:112]: This does not correspond to the rest of the sentence.
```suggestion
    /// be defined with View.setViewport(sf::FloatRect({ 0.f, 0.f }, { 0.5f, 1.f })).
```

Comment (Bromeon) [reply include/SFML/Graphics/View.hpp:112]: Also, we should probably use a SFML code style, i.e. lowercase variable names and no spaces (IIRC):
```cpp
view.setViewport(sf::FloatRect({0.f, 0.f}, {0.5f, 1.f}))
```

**Comment by ChrisThrasher** [comment]
Screenshot 2022-11-27 at 12 09 48 PM

I found some more places where old comments ought to get updated. Can you also fix these please?

**Comment by ChrisThrasher** [comment]
@Intemporel Are you able to fix the other places where the `FloatRect` doc comments are out of date?

**Comment by eXpl0it3r** [comment]
Any update on this @Intemporel?

**Comment by ChrisThrasher** [comment]
See #2421 for a more complete fix of this documentation issue.

**Pull request closed without merging**